benchmarking/benchmark_time_module.rb

```diff
@@ -0,0 +1,34 @@
+require_relative "benchmark_framework_module"
+include BenchmarkFrameworkModule
+
+require "benchmark"
+
+module BenchmarkTimeModule
+  TESTS = 1_000
+  class BenchmarkTime
```

Comment:
If you go with the pattern I mentioned earlier you could keep using your `BenchmarkFramework` module to scope to `Time`, `BenchmarkFramework::Time`